about.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Maenza Portfolio - About Page</title>
+<meta name="description" content="Resume for Martin Maenza" />
+<meta name="keywords" content="" />
+<meta http-equiv="author" content="Martin C. Maenza" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" type="text/css" href="styles/style0.css">
+</head>
+
+<body>
+<div class="wrapper">
+<header>
+<!-- page header with logo and navigation controls -->
+   <div class="logo">
+      <a href="index.html"><img src="images/MMportfolio.jpg" alt="Martin Maenza's Portfolio logo" /></a>
+   </div>
+   <nav>
+      <a href="about.html"><img src="images/about-button.jpg" alt="about button" /></a><br />
+      <a href="contact.html"><img src="images/contact-button.jpg" alt="contact button" /></a><br />
+      <a href="content.html"><img src="images/content-button.jpg" alt="content button" /></a><br />
+   </nav>
+</header>
+
+<div class="content">
+<p></p>
+<h1>Resume and Photograph</h1>
+<p>
+
+   <iframe width="500" height="400" src="resume.html"></iframe>
+
+   &nbsp;&nbsp;
```

Comment:
spacing deals with visual presentation and should therefore be implemented with css